server/configs/accounts.conf

```diff
@@ -41,8 +41,15 @@ accounts: {
     exports = [
       {service: "nats.time", response: stream}
       {service: "nats.photo", response: chunked}
-      {service: "nats.add", response: singleton, accounts: [cncf]}
       {service: "nats.sub"}
+      { service: "nats.add"
+        response: singleton
+        latency: {
+          sample: 100%
+          subject: "latency.tracking.add"
+        }
+        accounts: [cncf]
+      }
     ]
```

Comment:
I wasn't 100% sure if I should have added this here or if I should have created `latency.conf` or maybe some other way.

Reply:
Create a new one.

Reply:
Or lately we just create them on the fly and delete them.